dev/clint/src/clint/rules/do_not_disable.py

```diff
@@ -2,7 +2,7 @@
 
 
 class DoNotDisable(Rule):
-    DO_NOT_DISABLE = {"B006"}
+    DO_NOT_DISABLE = {"B006", "boolean-env-var"}
```

Comment:
The reason we need this rule is because we don't own ruff. For clint rules, we don't need to use the same approach.

Reply:
Ok